experimental/addons/kafka-operator/README.md

```diff
@@ -0,0 +1,249 @@
+kafka-topic                                                                                        kafka-cluster   10           1                    True
+```
+
+**Send and receive messages with Producer & Consumer**
+
+An event records the fact that "something happened" in the world or in your business. It is also called record or message in the documentation. When you read or write data to Kafka, you do this in the form of events. Conceptually, an event has a key, value, timestamp, and optional metadata headers. Here's an example event:
+
+- Event key: "Alice"
+- Event value: "Made a payment of $200 to Bob"
+- Event timestamp: "Jun. 25, 2020 at 2:06 p.m."
+
+Producers are those client applications that publish (write) events to Kafka, and consumers are those that subscribe to (read and process) these events. In Kafka, producers and consumers are fully decoupled and agnostic of each other, which is a key design element to achieve the high scalability that Kafka is known for. For example, producers never need to wait for consumers.
+
+Events are organized and durably stored in topics. Very simplified, a topic is similar to a folder in a filesystem, and the events are the files in that folder.
+
+*Make sure that `kafka-topic` and `kafka-cluster` got created successfullly before to move further because further steps utilizes `kafka-topic` and `kafka-cluster`.*
+
+With the cluster running, run a simple producer to send messages to the Kafka topic `kafka-topic`:
+
+```shell
+$ kubectl -n kafka-operator run kafka-producer -ti --image=quay.io/strimzi/kafka:0.33.2-kafka-3.4.0 --rm=true --restart=Never -- bin/kafka-console-producer.sh --bootstrap-server kafka-cluster-kafka-bootstrap:9092 --topic kafka-topic
+```
+
+And to receive messages in a different terminal from topic, run:
+
+```shell
+$ kubectl -n kafka-operator run kafka-consumer -ti --image=quay.io/strimzi/kafka:0.33.2-kafka-3.4.0 --rm=true --restart=Never -- bin/kafka-console-consumer.sh --bootstrap-server kafka-cluster-kafka-bootstrap:9092 --topic kafka-topic --from-beginning
+```
+
+**Create a kafka-bridge to enable HTTP connection to the kafka-cluster**
+
+Apply below YAML to create kafka bridge in `kafka-operator` namespace:
+
+```yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: kafka-bridge-sample
+spec:
+  components:
+    - type: kafka-bridge
+      name: kafka-bridge
+      properties:
+        replicas: 1
+        bootstrapServers: 'kafka-cluster-kafka-bootstrap:9092'
+        http:
+          port: 8080
+```
+
+after creating kafka-bridge, verify it.
+
+```shell
+$ kubectl get pods -n kafka-operator | grep kafka-bridge
+kafka-bridge-bridge-75549d4f89-c6qjf            1/1     Running   0          34m
+
+# Port forward the above pod
+$ kubectl port-forward -n kafka-operator kafka-bridge-bridge-75549d4f89-c6qjf 8080:8080
+
+# Visit on the port-forwarding port via CLI
+$ curl http://localhost:8080
+{"bridge_version":"0.24.0"}
```

Comment:
We should expose the bridge with a production usable way, such as using a ingress or service to expose it.

KubeVela is an application-centric platform, the end user of KubeVela don't have knowledge of kubectl. In this condition, how can the end user using this addon?

Reply:
Okay! @wonderflow I'll create a `nodePort` service for the bridge. and will try to replace use of `kubectl` with `vela` as much as possible to make this addon compatible with kubevela ecosystem :)